tests/test_get_pubkey.py

```diff
@@ -26,7 +26,8 @@ def pubkey_instruction_warning_approve(model: Firmware) -> Instructions:
     else:
         instructions.new_request("Unusual", NavInsID.USE_CASE_CHOICE_CONFIRM,
                                  NavInsID.USE_CASE_CHOICE_CONFIRM)
-        instructions.same_request("Confirm", NavInsID.USE_CASE_ADDRESS_CONFIRMATION_TAP,
+        instructions.same_request("Confirm", NavInsID.USE_CASE_ADDRESS_CONFIRMATION_TAP if model.name == "stax" 
```

Comment:
Is there really a difference in navigation in between `flex` and `stax` ?

Reply:
yes for Flex we need a swipe, to avoid to be stuck during address review